static/src/js/actions/focusedCollection.js

```diff
@@ -180,130 +184,155 @@ export const getFocusedCollection = () => async (dispatch, getState) => {
+        subscriberId: username
+      },
+      variableParams: {
+        limit: maxCmrPageSize
+      }
+    })
 
-  const response = graphQlRequestObject.search(graphQuery, {
-    params: {
-      conceptId: focusedCollectionId,
-      includeHasGranules: true,
-      includeTags: defaultCmrSearchTags.join(',')
-    },
-    subcriptionParams: {
-      subscriberId: username
-    }
-  })
-    .then((responseObject) => {
-      const payload = []
+    const payload = []
+
+    const {
+      data: responseData
+    } = response
 
+    const { data } = responseData
+    const { collection } = data
+
+    // If no results were returned, graphql will return `null`
+    if (collection) {
       const {
-        data: responseData
-      } = responseObject
-
-      const { data } = responseData
-      const { collection } = data
-
-      // If no results were returned, graphql will return `null`
-      if (collection) {
-        const {
-          abstract,
-          archiveAndDistributionInformation,
-          associatedDois,
-          boxes,
-          cloudHosted,
-          conceptId,
-          coordinateSystem,
-          dataCenter,
-          dataCenters,
-          duplicateCollections,
-          granules,
-          hasGranules,
-          nativeDataFormats,
-          relatedCollections,
-          services,
-          shortName,
-          subscriptions,
-          tags,
-          tilingIdentificationSystems,
-          title,
-          tools,
-          variables,
-          versionId
-        } = collection
-
-        // Look and see if there are any gibs tags
-        // If there are, check to see if the colormaps associated with the productids in the tags exists.
-        // If they don't we call an action to pull the colorMaps and add them to the metadata.colormaps
-        const gibsTags = tags ? getValueForTag('gibs', tags) : null
-        if (gibsTags && gibsTags.length > 0) {
-          const { product } = gibsTags[0]
-          dispatch(actions.getColorMap({ product }))
-        }
+        abstract,
+        archiveAndDistributionInformation,
+        associatedDois,
+        boxes,
+        cloudHosted,
+        conceptId,
+        coordinateSystem,
+        dataCenter,
+        dataCenters,
+        duplicateCollections,
+        granules,
+        hasGranules,
+        nativeDataFormats,
+        relatedCollections,
+        services,
+        shortName,
+        subscriptions,
+        tags,
+        tilingIdentificationSystems,
+        title,
+        tools,
+        variables,
+        versionId
+      } = collection
 
-        // Formats the metadata returned from graphql for use throughout the application
-        const focusedMetadata = createFocusedCollectionMetadata(
-          collection,
-          authToken,
-          earthdataEnvironment
+      // Retrieves all variables if there are more than `maxCmrPageSize`
+      if (variables && variables.count && variables.count > maxCmrPageSize) {
```

Comment:
any way we can simplify this conditional? From what I can see should just be able to use two assertions since if there are variables from the response it will include count
```suggestion
      if (variables && variables.count > maxCmrPageSize) {
```